frontend/app/templates/create-badges.hbs

```diff
@@ -92,6 +92,7 @@
                         </div>
 
                     </div>
+
```

Comment:
please do no add empty lines here.
please make the changes so that we can merge this.
cheers! :+1: